app_backend/user.py

```diff
@@ -0,0 +1,20 @@
+class User():
+
+    """
+    User keeps track of the quack app users balanace information
+
+    name: str
+        The name of the user
+
+    dooley_dollars: float
+        The current balance of dooley dollars
+
+    meal_swipes: int
+        The current amount of remaining meal swipes
+
+    """
+
+    def __init__(self, name, dooley_dollars, meal_swipes):
```

Comment:
Instead of name consider using UUID. This is in preparation for our data management with the database

Reply:
@RafaelPiloto10  should the variable UUID be in all caps?

Reply:
Great question, I think lowercase to remain consistent